common/models/wordbreakers/src/default/data.ts

```diff
@@ -4,7 +4,7 @@ export namespace data {
 /**
  * Valid values for a word break property.
  */
-export const enum WordBreakProperty {
+export const enum WordBreakProperty { // Scary bit:  this does not exist as an object at run-time!
```

Comment:
```suggestion
export const enum WordBreakProperty { 
```

I don't understand the point of that comment? That's a standard feature of TypeScript.

Reply:
I noted this in a later comment, but since it's relevant to my comment above:  the minified propertyMap used for string -> numeric property ID lookup... is 206 bytes minified by itself. 

_Then_, there's the two methods that exist to map string-based specifications to the internal int/enum-based one:
- `propertyMatch`:  minifies at 260 bytes
- `propertyVal`: minifies at 250 bytes(!) - I didn't expect this one to be as large as it apparently is; looks like that's due to optional chaining.